src/collective/lazysizes/profiles.zcml

```diff
@@ -2,15 +2,36 @@
     xmlns="http://namespaces.zope.org/zope"
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
     xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
     i18n_domain="collective.lazysizes">
 
   <genericsetup:registerProfile
+      name="base"
```

Comment:
can't we use this new feature instead? https://github.com/plone/Products.CMFPlone/issues/1406

Reply:
Ha, cool - I actually didn't know this.
But still - I'd prefer to not import the jsregistry.xml.
On the other hand, this might not hurt that much...

Reply:
I prefer to move the registration of JS in Plone 4 to a setup handler in Python code then because this is causing other issues as you can see on tests.